indico/web/flask/blueprints/event/creation.py

```diff
@@ -13,31 +13,23 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with Indico; if not, see <http://www.gnu.org/licenses/>.
-from flask import url_for
-from werkzeug.utils import redirect
 
 from MaKaC.webinterface.rh import categoryDisplay
 from indico.web.flask.wrappers import IndicoBlueprint
 
 
-def _redirect_simple_event(**kwargs):
-    # simple_event is confusing so we always use "lecture" in the URL
-    return redirect(url_for('.conferenceCreation', event_type='lecture', **kwargs))
-
-
-event_creation = IndicoBlueprint('event_creation', __name__, url_prefix='/event/create')
+event_creation = IndicoBlueprint('event_creation', __name__, url_prefix='/event/create-old')
```

Comment:
Didn't mean the underscore if we are supposed to keep the old endpoint. I guess we are getting rid of it before the release.